packages/web/src/pages/profile-page/ProfilePageProvider.tsx

```diff
@@ -267,6 +269,10 @@ class ProfilePage extends PureComponent<ProfilePageProps, ProfilePageState> {
     this.setState({ areArtistRecommendationsVisible: false })
   }
 
+  onCloseInboxUnavailableModal = () => {
+    this.setState({ showInboxUnavailableModal: false })
```

Comment:
RE: modals @sliptype / @dylanjeffers / @schottra - Do we hate this?

(Modal mounted in desktop version below)